samples/guide/src/main/java/okhttp3/recipes/BasicAuth.java

```diff
@@ -0,0 +1,58 @@
+package okhttp3.recipes;
+
+import okhttp3.Credentials;
+import okhttp3.Interceptor;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.IOException;
+
+public class BasicAuth {
```

Comment:
Sicne this is so close to Authenticate sample, I suggest documenting the difference here (peremptive etc).

Reply:
Maybe rename to PreemptiveAuth ?